config/locales/en/providers.yml

```diff
@@ -838,10 +838,6 @@ en:
     has_national_insurance_numbers:
       show:
         page_title: Does your client have a National Insurance number?
-        hint: |
```

Comment:
I wonder if this was a typo, should the switch have been on proceeding.special_children_act/non_means_tested rather than setting.special_children_act?  Is it worth checking with design?
It just feels like this hint should be shown for DA, PLF etc

Reply:
Double checked with them and it's good to be removed for everything